DQM/BeamMonitor/plugins/BeamConditionsMonitor.h

```diff
@@ -36,7 +36,7 @@ class BeamConditionsMonitor : public edm::EDAnalyzer {
   void beginJob() override;
 
   // BeginRun
-  void beginRun(const edm::Run& r, const edm::EventSetup& c) override;
+  // void beginRun(const edm::Run& r, const edm::EventSetup& c) override;
```

Comment:
Should these be removed for good? (also the `endRun`)

Reply:
yup I forgot to remove them after testing